starter-code/Portfolio.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+
+    <head>
+        <title>Rawan Alrajeh | Portfolio</title>
+        <link rel="stylesheet" href="../starter-code/Css/">
+    </head>
+<body>
+    <p>&nbsp;</p>
+<header>
+<div class="isheader" style="text-align: center;"><span style="color: #666699;"><strong><a style="color: #666699;" href="../starter-code/About Page.html">About Me</a> | <a style="color: #666699;" href="Rweb2.html">Portfolio</a></strong></span></div>
```

Comment:
be careful with file naming and locations, specially with spaces as they can cause errors in the future.